src/pages/lessons/[slug].tsx

```diff
@@ -297,7 +299,17 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
   React.useEffect(() => {
     send('LOAD')
     setLesson(initialLesson)
-    loadLesson(initialLesson.slug, getAccessTokenFromCookie()).then(setLesson)
+
+    async function run() {
```

Comment:
the `then` call was acting extremely strangely and I didn't figure out a technical reason why, but this works as expected